client/app/iplb/modal/offer/iplb-offer-change.html

```diff
@@ -1,18 +1,17 @@
-<cui-modal>
-    <form id="IplbOfferChangeForm">
-        <cui-modal-header data-on-dismiss="IpLoadBalancerOfferChangeCtrl.cancel()"></cui-modal-header>
-        <cui-modal-body data-title=":: 'iplb_modal_offer_change_title' | translate">
-            <oui-spinner data-ng-if="IpLoadBalancerOfferChangeCtrl.isModalLoading()"></oui-spinner>
-            <div data-ng-if="!IpLoadBalancerOfferChangeCtrl.isModalLoading()">
-                Old offer => {{ IpLoadBalancerOfferChangeCtrl.configuration.data.offer }}
-                <cui-modal-text data-text="'EDIT SOME OFFER HERE'"></cui-modal-text>
-            </div>
-        </cui-modal-body>
-        <cui-modal-footer>
-            <cui-cancel-button data-text="'common_cancel' | translate" data-on-click="IpLoadBalancerOfferChangeCtrl.cancel()"></cui-cancel-button>
-            <cui-submit-button data-text="'common_edit' | translate"
-                data-on-click="IpLoadBalancerOfferChangeCtrl.confirm()"
-                data-disabled="IplbOfferChangeForm.$invalid || IpLoadBalancerOfferChangeCtrl.isModalLoading()"></cui-submit-button>
-        </cui-modal-footer>
-    </form>
-</cui-modal>
+<form id="IplbOfferChangeForm"
+      name="IplbOfferChangeForm"
+      data-ng-submit="IplbOfferChangeForm.$valid && IpLoadBalancerOfferChangeCtrl.confirm()"
+      novalidate>
+    <oui-modal
+        data-title="{{ ::'iplb_modal_offer_change_title' | translate }}"
+        data-primary-label="{{ ::'common_edit' | translate }}"
+        data-secondary-action="IpLoadBalancerOfferChangeCtrl.cancel()"
+        data-secondary-label="{{ ::'common_cancel' | translate }}"
+        data-on-dismiss="IpLoadBalancerOfferChangeCtrl.cancel()"
+        data-loading="IpLoadBalancerOfferChangeCtrl.isModalLoading()">
+        <p class="oui-modal__text">
```

Comment:
Should be remove, right?

Reply:
The modal is not completely implemented and is not shown. Have just changed cui-modal to oui-modal here.